src/components/step-wrapper/step-wrapper.styles.js

```diff
@@ -14,7 +16,8 @@ export const styles = {
     color: 'text',
     fontWeight: 600,
     borderBottom: '3px solid',
-    pb: '14px'
+    pb: '14px',
+    animation: `${fade} 0.5s ease-in-out`
```

Comment:
It is used in 4 places. It would be best if you made it general.

Reply:
fixed